core-bundle/src/EventListener/DataContainer/PageSearchListener.php

```diff
@@ -32,7 +32,7 @@ public function __construct(private ContaoFramework $framework, private Connecti
      */
     public function onSaveAlias(string $value, DataContainer $dc): string
     {
-        if ($value === $dc->activeRecord->alias) {
+        if ($value === $dc->getCurrentRecord()['alias']) {
```

Comment:
`$dc->getCurrentRecord()` can return `null`. Shouldn‘t we handle this?

Reply:
I didn't cover these cases because the current code didn't either. We can still do that later if there's any case but apparently there never has been because accessing `->alias` on `null` would also have caused an error.